### ISSUE TITLE
[BUG] Name of list in a DC device is shown in English on the tiles. 

### ISSUE THREAD

**Issue by jeesnl** [mennovanhout/flutter-hdashboards-issues#162]
**Describe the bug**
When a DC device is created with a list capebility it’s shown in English in the tile. But in long push screen it’s shown in Dutch. 

**To Reproduce**
Create a DC device with a list capability 
Add that device to HD. 

**Expected behavior**
Capability descriptions should be in the user language. 

**Screenshots**
![IMG_9629](https://github.com/mennovanhout/flutter-hdashboards-issues/assets/39216786/6f26e8d5-ffa1-447f-8405-093709182074)
![IMG_9630](https://github.com/mennovanhout/flutter-hdashboards-issues/assets/39216786/e240d613-b223-4085-b47b-1382b147ff06)

**Comment by StaffordBully** [comment]
When added a list, it's shown in you Homey language. So can it be your Homey is setup in Dutch?

**Comment by jeesnl** [comment]
Homey and HD are both Dutch.

**Comment by StaffordBully** [comment]
Hdashboards is in English

**Comment by jeesnl** [comment]
You’re right, but the long press screen shows it in Dutch. So in the tile it should also be Dutch.

**Comment by mennovanhout** [comment]
Fixed in 0.5.2
